keyboards/controllerworks/mini42/rules.mk

```diff
@@ -0,0 +1,13 @@
+LTO_ENABLE = yes
```

Comment:
Most of these can be moved to the info.json (everything but the "driver" option)

Reply:
I have now moved everything I could find into the json file.